misc/did-jwt/src/did-jwt.ts

```diff
@@ -0,0 +1,88 @@
+import { base58btc } from "multiformats/bases/base58";
+import bs58 from "bs58";
+import * as ed25519 from "@noble/ed25519";
+import { JwtHeader, JwtPayload } from "./types";
+import { concatUInt8Arrays, makeBase64UrlSafe, objectToHex } from "./helpers";
+import {
+  DID_DELIMITER,
+  DID_METHOD_PKH,
+  DID_PREFIX,
+  DAY_IN_MS,
+  JWT_DELIMITER,
+  MULTICODEC_ED25519_HEADER,
+  DID_METHOD_KEY,
+  MULTICODEC_X25519_HEADER,
+} from "./constants";
+
+export const composeDidPkh = (accountId: string) => {
```

Comment:
Very useful! I had to compose `did:pkh:..` in several repos so this will come in very handy.
Might be a good idea to add `extractAddressFromDidPkh` or similar in the future that returns the address too